### ISSUE TITLE
Right ventricular thrombus

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11103]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
A blood clot (thrombus) localized in the Right ventricle (RV).
PMID: 28032368 In addition to the characteristic severe dilatation of the right heart chambers and absence of the RV myocardium, there may be coexistent **RV aneurysms and thrombus**, and other associated cardiac anomalies such as pulmonary atresia and tricuspid valve malformation.

PMID: 33117927 The diagnosis of Uhl’s anomaly was established, and the patient started diuretics with reasonable control of HF symptoms. However,
during the 3 years after diagnosis, he presented multiple decompensations of HF, requiring hospitalization. Additionally, he was diagnosed with atrial fibrillation and **thrombus** in the **right atrium and ventricle**.

PMID: 33750225 Cardiac MRI unenhanced short-axis (Fig 2, A) and four-chamber cine steady-state free precession imaging (Fig 2, B) showed severe enlargement of the RV; the RV free wall myocardium was extremely thinned, the apical trabeculation was almost absent, and the wall was irregular with a parchmentlike appearance. **Thrombus was seen in the RV outflow tract.**

**Parent term (use hpo.jax.org/app)**
Abnormal thrombosis HP:0001977

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Uhl anomaly  ORPHA:3403

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001282
- label: Right ventricular thrombus
- parents: [Abnormal right ventricle morphology (HP:0001707)](https://hpo.jax.org/app/browse/term/HP:0001707)
- pmid: PMID:38746065
- def: A blood clot (thrombus) localized in the right ventricle.
- comment: Right ventricular thrombus is a rare finding found in 4% of people diagnosed with pulmonary embolism. Although right ventricular thrombi are usually associated with atrial fibrillation, deep venous vein thrombi, or intracardiac procedures, isolated right ventricular thrombi are rare. Right ventricular thrombus has also been reported in patients with right ventricular infarction, as hypokinesis of the right ventricle causes blood stasis and promotes thrombosis as per Virchow's triad.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11103
- synonyms:
